GuessNum1A2B/src/main/kotlin/tw/waterballsa/utopia/GuessNum1A2B/listener.kt

```diff
@@ -0,0 +1,44 @@
+package tw.waterballsa.utopia.GuessNum1A2B
+import ch.qos.logback.core.util.OptionHelper.getEnv
+import com.mongodb.annotations.Beta
+import mu.KotlinLogging
+import net.dv8tion.jda.api.JDA
+import net.dv8tion.jda.api.events.message.MessageReceivedEvent
+import net.dv8tion.jda.api.events.session.ReadyEvent
+import tw.waterballsa.utopia.GuessNum1A2B.domain.GuessNum1A2B
+import tw.waterballsa.utopia.GuessNum1A2B.domain.generateSecretNumber
+import tw.waterballsa.utopia.commons.config.WsaDiscordProperties
+import tw.waterballsa.utopia.jda.listener
+
+val logger = KotlinLogging.logger{}
+val GuessNumChannelID = 1093177540346130453
```

Comment:
Why is the first letter of the variable name capitalized ? 
Please follow the naming convention. 
Also, this should be placed in the yml file.